src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -1,8 +1,9 @@
 package seedu.address.logic;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static seedu.address.commons.core.Messages.MESSAGE_INVALID_MODULE_DISPLAYED_INDEX;
-import static seedu.address.commons.core.Messages.MESSAGE_UNKNOWN_COMMAND;
+// import static seedu.address.commons.core.Messages.MESSAGE_INVALID_MODULE_DISPLAYED_INDEX;
```

Comment:
I think it is better to delete them instead of having them commented out if they are not used

Reply:
The reason I commented out is just to prevent us from forgetting which functions need to be imported. 
But if there's really no need for these we can simply delete them!